src/components/app.tsx

```diff
@@ -32,6 +36,10 @@ export const App = () => {
   const playingRef = useRef<boolean>();
   playingRef.current = playing;
 
+  useAutoWidth({
+    container: document.body,
+    onWidthChange: (newWidth) => setGraphWidth(newWidth - 2 * GRAPH_MARGIN)
```

Comment:
Since we're creating this dynamically as a lambda function, it seems like this would result in re-running the `useEffect()` in `useAutoWidth()` on every render. I would think that `useCallback()` would make sense here.